src/components/Login/Login.test.js

```diff
@@ -3,11 +3,11 @@ import Login from './Login';
 
 test('renders learn react link', () => {
   render(<Login/>);
-  const linkElement = screen.getByText(/Login/i);
+  const linkElement = screen.getByText(/Email/i);
   expect(linkElement).toBeInTheDocument();
-  const email = screen.getByPlaceholderText(/email/i);
+  const email = screen.getByPlaceholderText(/Email address/i);
   expect(email).toBeInTheDocument();
-  const password = screen.getByPlaceholderText(/password/i);
+  const password = screen.getByPlaceholderText(/••••••••••/i);
```

Comment:
•••••••••• placeholder is not very descriptive. User's input should not be clear text but placeholder should advice user to type correct input